sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/model/ValueMappingModel.java

```diff
@@ -36,13 +36,19 @@ public ValueMappingModel setApiToImpl(List<ValueMapping> apiToImpl) {
   }
 
   public Optional<String> mapToImpl(String apiValue) {
+    if (apiValue == null) {
+      return Optional.empty();
+    }
     return Optional.ofNullable(
         apiToImpl.getOrDefault(
             apiValue.toUpperCase(Locale.ROOT),
             apiToImplBidirectional.get(apiValue.toUpperCase(Locale.ROOT))));
   }
 
   public Optional<String> mapToApi(String implValue) {
+    if (implValue == null) {
+      return Optional.empty();
+    }
```

Comment:
This condition is [not covered](https://sonarcloud.io/component_measures?metric=new_lines_to_cover&selected=SDA-SE_sda-dropwizard-commons%3Asda-commons-server-key-mgmt%2Fsrc%2Fmain%2Fjava%2Forg%2Fsdase%2Fcommons%2Fkeymgmt%2Fmodel%2FValueMappingModel.java&view=list&id=SDA-SE_sda-dropwizard-commons). If `null` values already caused trouble, I would prefer a test case.